build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -651,6 +651,17 @@ message Command {
   // property is not recognized by the server, the server will return an
   // `INVALID_ARGUMENT`.
   repeated string output_node_properties = 8;
+
+  // True if the server should interpret output_paths as being relative to the
+  // input root. False if the server should interpret output_paths as being
+  // relative to the working directory. This is intended to provide a migration
+  // path from the old (working_directory) to the new (input root) semantics for
+  // output paths; eventually this field will be deprecated and ignored by the
+  // server.
+  //
+  // Note that this field does NOT affect the semantics of the output_files and
+  // output_directories fields.
```

Comment:
If the goal is for servers to ignore this field after a transition period, wouldn't it make more sense to have this field affect the semantics of all `output_*` fields such that servers can remove the old (working directory-relative) logic? Or do you expect servers to also drop support for the deprecated `output_files` and `output_directories` at that point?

Reply:
I think the reasoning is that if clients are going to be patched up to set the boolean flag, they might as well be adjusted to use `output_paths`. That implies that if servers drop support for the old semantics, they can drop support for `output_files` and `output_directories` at the same time.

Reply:
Exactly. I suspected the appetite for patching up a field that's already deprecated would be low, and this would provide a nudge to migrate to output_paths. I'm certainly willing to reconsider that position.

Reply:
Having server implementations drop support for `output_files` and `output_directories` at the same time as (or earlier than) this new flag makes sense (incrementing `deprecated_api_version`), so no objections from me. I also see no issues from the BuildBox (BuildGrid worker) implementation perspective.